apps/ejabberd/src/mod_privacy.erl

```diff
@@ -336,7 +336,7 @@ check_packet(Acc, User, Server,
             Type = xml:get_attr_s(<<"type">>, Packet#xmlel.attrs),
             check_packet_aux(List, PType, Type, LJID, Subscription, Groups)
     end,
-    mongoose_acc:put(privacy_check, CheckResult, Acc).
+    mongoose_acc:store(privacy_check, jid:to_lower(To), CheckResult, Acc).
```

Comment:
Could you explain why exactly do we need to introduce a nested map here? It's not clear to me.